cmake/LYTestWrappers.cmake

```diff
@@ -261,9 +261,13 @@ function(ly_add_test)
     # Check to see whether or not this test target has been stored in the global list for walking by the test impact analysis framework
     get_property(all_tests GLOBAL PROPERTY LY_ALL_TESTS)
     if(NOT "${test_target}" IN_LIST all_tests)
+        # Extract the test target name from the namespace::target_name composite
+        string(REGEX REPLACE ".*::" "" test_name "${test_target}")
+        # Store the test target name sans namespace so they can be looked up without the preceeding namespace
+        set_property(GLOBAL APPEND PROPERTY O3DE_ALL_TESTS_DE_NAMESPACED ${test_name})
```

Comment:
This should use moved to a function similar to `ly_de_alias_target` to `ly_de_alias_test`